backend/src/main/java/team15/airbnb/domain/accommodation/Accommodation.java

```diff
@@ -0,0 +1,91 @@
+package team15.airbnb.domain.accommodation;
+
+import com.sun.istack.NotNull;
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.List;
+import javax.persistence.Column;
+import javax.persistence.Embedded;
+import javax.persistence.Entity;
+import javax.persistence.EnumType;
+import javax.persistence.Enumerated;
+import javax.persistence.FetchType;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import javax.persistence.OneToMany;
+import javax.persistence.OneToOne;
+import team15.airbnb.domain.BaseEntity;
+import team15.airbnb.domain.Reservation;
+import team15.airbnb.domain.Review;
+import team15.airbnb.domain.User;
+
+@Entity
+public class Accommodation extends BaseEntity {
+
+	@Id
+	@Column(name = "accommodation_id")
+	@GeneratedValue(strategy = GenerationType.IDENTITY)
+	private Long id;
+
+	@Column(length = 50)
+	private String accommodationName;
```

Comment:
name 정도로 가도 괜찮지않을까요? ㅎㅎ
```suggestion
	private String name;
```

Reply:
테이블 칼럼 명은 `accommodation_name`으로, Entity의 필드명은 `name`으로 하는 것으로 다른 데이터들도 통일하려 합니다.

이유 : ex. `accommodation.getId()` 일 때가 `accommodation.getAccommodationId()` 보다 가독성이나 의미 면에서 더 명확해보이기 때문입니다.